KNOWN_ISSUES.md

```diff
@@ -13,5 +13,8 @@
 
 # Networking
 
+- Some network ranges are not allowed for node group usage, so please not use the following when creating a private network for the cluster:
+  - `10.0.0.0/8`
```

Comment:
This is `10.<WHATEVER>` now, right? Should this be `10.0.<WHATEVER>`, i.e. `10.0.0.0/16`?

Reply:
Checked! `10.0.0.0/8` is good to go.